ZCS-1417

mihaipopahtp opened the pull request:
- adding Vietnamese translation

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Zimbra/zm-admin-ajax?pullRequest=4) <br/>All committers have signed the CLA.

silentsakky commented on WebRoot/js/ajax/config/msgs/AjxMsg_vi.properties:
```diff
@@ -0,0 +1,157 @@
+# 
```
file location looks wrong

mihaipopahtp replied:
It was, sorry. 
I removed it and rebased. I only see the 3 correct files now. Hope you do too.
https://github.com/Zimbra/zm-ajax/pull/34/files

silentsakky committed:
ZCS-1417
- adding Vietnamese translation
- removing WebRoot\js\ajax\config\msgs\AjxMsg_vi.properties
- adding ZCS-2592 updates to smart quotes + minor fixes
- changes after linguistic testing
- updating capitalalization for month names
